macros/anaUltraThreshold.py

```diff
@@ -1,4 +1,4 @@
-#!/bin/env python
+#!/bin/env python2.7
```

Comment:
only if run on a machine without ```python2.7``` (post ```cc7``` switchover this won't be a problem), in which case, it should simply not run.
for ultimate portability, consider checking the python version, and initializing the dict depending on the version:
https://stackoverflow.com/questions/1747817/create-a-dictionary-with-list-comprehension-in-python#1747827